test/unit/utils/BaseFactory.mocha.ts

```diff
@@ -0,0 +1,69 @@
+/// <reference path='../test.mocha.ts' />
+
+class BaseFactoryStub extends n3Charts.Utils.BaseFactory {
```

Comment:
This should be named FactoryStub, I think, because you're not stubbing the BaseClass but a class that inherits from it.